crates/api/src/deploy.rs

```diff
@@ -90,8 +94,19 @@ impl DeployFile {
     }
 
     /// Returns an iterator over mapped container volumes.
-    pub fn vols(&self) -> impl Iterator<Item = &deploy::ContainerVolume> {
-        self.vols.iter()
+    pub fn public_vols<'a>(&'a self) -> impl Iterator<Item = deploy::ContainerVolume> + 'a {
```

Comment:
This is a breaking change that warrants a bump in the crate's version.

Reply:
Also, the docs should be updated for this method.

Reply:
fixed.

Reply:
Could you also bump the version of the crate itself to `0.3.0`? Otherwise, if we push as `0.2.3` for instance, this will break `gfaas` for me, etc.

Reply:
??

```
gfaas $ cargo tree | grep ya-runtime
├── ya-runtime-wasi v0.2.2 (https://github.com/golemfactory/ya-runtime-wasi.git?branch=prekucki/sgx-profile#909cef13)
│   ├── ya-runtime-api v0.1.0 (https://github.com/golemfactory/yagna.git#344757c7)
gfaas $ cargo check
    Finished dev [unoptimized + debuginfo] target(s) in 0.17s
$ git diff -w
diff --git a/Cargo.toml b/Cargo.toml
index 31b3518..8b29a92 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -19,7 +19,7 @@ zip = "0.5"
 serde_json = "1"
 tempfile = "3.1"
 tokio = { version = "0.2", features = ["blocking"] }
-ya-runtime-wasi = "0.2"
+ya-runtime-wasi = { version = "0.2", git= "https://github.com/golemfactory/ya-runtime-wasi.git", branch="prekucki/sgx-profile" }
 yarapi = "0.1"
 ya-agreement-utils = "0.1"
 thiserror = "1.0.20"

```

Reply:
I'm not sure where the confusion lies. All I'm asking is to bump the version of `ya-runtime-wasi` crate since you've changed the public API thus breaking the compat from the previously released versions, am I correct?

Reply:
No, we are not breaking compatibility. 

```rust
    #[deprecated = "use [`public_vols`](#method.public_vols) or [`container_vols`](#method.container_vols) instead"]
    pub fn vols(&self) -> impl Iterator<Item = &deploy::ContainerVolume> {
        self.container_vols()
    }
```

Reply:
Ah, nice! Somehow I missed that one! Thanks for pointing this one out to me, and apologies for confusion!